ci/scripts/autobump-dependencies.py

```diff
@@ -309,37 +320,34 @@ def main() -> None:
             "LIBPCAP_VERSION",
             "1.10",
             "https://www.tcpdump.org/release/",
-            package="pcap-api",
-        ),
-        WebLinkDependency(
-            "libpcap",
-            "LIBPCAP_VERSION",
-            "1.10",
-            "https://www.tcpdump.org/release/",
-            package="pcap-agent",
+            packages=["pcap-api", "pcap-agent"],
         ),
     ]
 
     for dependency in dependencies:
-        current_version = dependency.current_version
-        latest_release = dependency.latest_release
-        latest_version = latest_release.version
-
-        if latest_version <= current_version:
-            print(f"[{dependency.name}] already on the latest version: {latest_version} " f"(pinned: {dependency.pinned_version}.*)")
-            continue
-
-        if dependency.open_pr_exists():
-            print(f"[{dependency.name}] Open bump PR exists (for branch: {dependency.pr_branch})")
-            continue
-        print(f"[{dependency.name}] Version-Bump required: {current_version} --> {latest_version}")
-        latest_release.download()
-        dependency.remove_current_blob()
-        latest_release.add_blob()
-        dependency.update_packaging_file()
-        if not DRY_RUN:
-            write_private_yaml()
-            BoshHelper.upload_blobs()
+        for package in dependency.packages:
+            current_version = dependency.check_current_version(package)
+            latest_release = dependency.latest_release
+            latest_version = latest_release.version
+
+            if latest_version <= current_version:
+                print(f"[{dependency.name}] in {package} already on the latest version: {latest_version} " f"(pinned: {dependency.pinned_version}.*)")
+                continue
+
+            if dependency.open_pr_exists():
```

Comment:
I think the PR should be checked before iterating through the packages, as we want one PR per dependency, right?

Reply:
Nevermind. As discussed, this approach is fine.